helpers.ts

```diff
@@ -90,27 +80,25 @@ export function getParsedOptions(options: any, shorthands: any, clientName: stri
 
 export function validateYargsArguments(parsed: any, knownOpts: any, shorthands: any, clientName?: string): void {
 	if(path.basename(process.argv[1]).indexOf(clientName) !== -1) {
+		var errors = $injector.resolve("$errors");
```

Comment:
It looks like errors cannot be resolved here. The problem is that bootstrap is resolved before starting the fiber and before the injector is created (from appbuilder-cli.ts: 
```Typescript
require("./bootstrap");
import fiberBootstrap = require("./fiber-bootstrap");

fiberBootstrap.run(() => {
```
So when options is required in bootstrap, it calls helpers.getParsedOptions, which calls this method. We try to resolve errors, which depends on logger and config. Configuration's constructor has wait, so we receive can't wait without a fiber as we are not in fiberBootstrap's run.